content/en/security/application_security/threats/library_configuration.md

```diff
@@ -33,7 +33,11 @@ ASM automatically attempts to resolve `http.client_ip` from several well-known h
 
 ## Track authenticated bad actors
 
-Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions. Read [Tracking User Activity][1] for more information.
+Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions.
+
+The Datadog Tracing Library will attempt to auto-detect user login and signup events when it detects compatible authentication frameworks in use.
+
+Read [Tracking User Activity][1] for more information on how to manually track user activity, or if you wish to opt out - [Disabling automatic user activity event tracking][7].
```

Comment:
Opt out? 

Did you men opt-in? 

Feels weird to mention opt-out

Reply:
It is an `opt-out`, because the autoinstrumentation is on by default (`safe`).